### ISSUE TITLE
Center Loss backprop optimzation

### ISSUE THREAD

**Issue by wenfahu** [jxgu1016/MNIST_center_loss_pytorch#2]
What about using scatter op in pytorch to replace the for loop in center loss backward.
There is an undocumented op `torch.Tensor.scatter_ add_`

**Comment by jxgu1016** [comment]
@wenfahu Could you accomplish this optimization and pull a request? I am quite busy now and have no free hands on this...
